lib/index.js

```diff
@@ -178,6 +178,34 @@ function _ensure_cheerio(value) {
     return value;
 }
 
+class DictionaryAsset extends BaseAsset {
+    constructor() {
+        super();
+        this._object_type = "DictionaryObject";
+        this._content_type = "text/html";
```

Comment:
Sounds good - there's a `render` method now in the DictionaryAsset class which sets the relevant fields (word, wordDefinition, partOfSpeech) to already existing fields on the Blog template.